src/applications/gi/reducers/calculator.js

```diff
@@ -89,7 +132,12 @@ export default function (state = INITIAL_STATE, action) {
         tuitionFees: tuitionInState || 0,
         inStateTuitionFees: tuitionInState || 0,
         books: books || 0,
-        calendar: calendar || 'semesters'
+        calendar: calendar || 'semesters',
+        yellowRibbonDegreeLevel: yellowRibbonPrograms[0].degreeLevel,
+        yellowRibbonDivision: yellowRibbonPrograms[0].divisionProfessionalSchool,
+        yellowRibbonDegreeLevelOptions,
+        yellowRibbonDivisionOptions,
+        yellowRibbonPrograms
```

Comment:
this is the new `yellowRibbonPrograms` attribute that's part of the institution entity. 

I tried to keep this attribute in the `profile` leaf of the state since that's where the rest of the institution info is, but this information is needed in the same context with the input values to recalculate and set some of the select options. 

The inputs use a single reducer to handle their state which has coupled this state to the UI.